consensus/hotstuff/committee.go

```diff
@@ -112,7 +117,7 @@ type DynamicCommittee interface {
 	//   * contains no duplicates.
 	// The list of all legitimate HotStuff participants for the given epoch can be obtained by using `filter.Any`
```

Comment:
I think part of this documentation is outdated, as we don't have a "selector" anymore

Suggestion:


1.  remove line 118
     ```suggestion
     ```
2. remove [line 112](https://github.com/onflow/flow-go/blob/530e2d602eb8a72dbadcfd7f6d81b7076604c732/consensus/hotstuff/committee.go#L112)